src/vivarium_gates_nutrition_optimization_child/constants/data_values.py

```diff
@@ -150,7 +150,7 @@ class __LBWSG(NamedTuple):
 ############################
 class __Wasting(NamedTuple):
     # Wasting age start (in years)
-    START_AGE: float = 0.5
+    DYNAMIC_START_AGE: float = 0.5
```

Comment:
Why is `data_values.WASTING` defined twice? This second one is just shadowing the first. Are they identical?

Reply:
I think this happened when I accidentally copied these separately into the Artifact and Implementation tickets, in slightly different places. The data are all the same, apart from a diarrhea vicious cycle parameter that isn't being used. De-duping.